### ISSUE TITLE
Using add_style style index on add_row gives error

### ISSUE THREAD

**Issue by westonganger** [axlsx-styler-gem/axlsx_styler#36]
As mentioned in https://github.com/axlsx-styler-gem/axlsx_styler/issues/14#issuecomment-1256271657

For `no implicit conversion of Integer into Hash` this is another error I was wondering how didnt get triggered. Seemed you failed to mention this. I can make a separate PR to resolve this likely.

```ruby
# Not working
border = s.add_style border: { style: :thin, color: "000000" }
sheet.add_row ['1', '2', '3', '4'], style: border
```

```
no implicit conversion of Integer into Hash
```

Seems we need to modify the worksheet#add_styles method to handle passing of style index (style number / style object)

**Comment by westonganger** [comment]
Closing as per https://github.com/axlsx-styler-gem/axlsx_styler/pull/37#issuecomment-1259851462
